cmd/cvp-trigger/main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/client-go/rest"
+
+	pjapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/interrupts"
+	"k8s.io/test-infra/prow/pjutil"
+
+	"github.com/openshift/ci-tools/pkg/rehearse"
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+const (
+	prowConfigPathOption = "prow-config-path"
+	jobConfigPathOption  = "job-config-path"
+	periodicOption       = "periodic"
+)
+
+type options struct {
+	dry bool
+
+	prowConfigPath string
+	jobConfigPath  string
+
+	periodic string
+}
+
+func gatherOptions() options {
+	o := options{}
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+
+	fs.BoolVar(&o.dry, "dry-run", true, "Whether to actually submit the job to Prow")
```

Comment:
nit: better UX is to make the user provide a positive, so make a user pass `--dry-run=true` for a program that defaults to doing something and `--confirm=true` for something that defaults to doing nothing

Reply:
Yep, I had a real dry first, then changed it because I like the "safe by default" pattern. I'll change this to `--confirm` with a `false` default.